### PR TITLE
Release/6.20.3

### PR DESCRIPTION
**Pull request by InnocentSmasher**
Releasing accordion fix into production: https://github.com/waynestate/base-site/pull/490

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "base",
   "private": true,
-  "version": "6.20.1",
+  "version": "6.20.3",
```
Comment (InnocentSmasher) [inline package.json:4]: I forgot to bump the version for the last hotifx. I was rushing.

Comment (chrispelzer) [reply package.json:4]: It happens. Just remember for next time.

**Review by chrispelzer — COMMENTED** [review]
All good

**Review by nickdenardis — COMMENTED** [review]
👍

**Pull request closed without merging**